ci_scripts/ci_post_clone.sh

```diff
@@ -74,7 +74,8 @@ $QT_IOS_PATH/bin/qt-cmake -S . -GXcode \
   -DCMAKE_OSX_ARCHITECTURES="arm64" \
   -DSENTRY_DSN=$SENTRY_DSN \
   -DSENTRY_ENVELOPE_ENDPOINT=$SENTRY_ENVELOPE_ENDPOINT \
-  -DCMAKE_BUILD_TYPE=Release
+  -DCMAKE_BUILD_TYPE=Release \
+  -DMVPN_IOS_ADJUST_TOKEN=$MVPN_IOS_ADJUST_TOKEN
```

Comment:
I think you have tried to fix it twice, which is making the expectation of how to enable Adjust a little confusing here. It appears to be set as an environment variable by the CI, and here we are also passing it as a variable definition to CMake with the same name.

I would recommend that we go back to using a CMake variable definition for the token and remove the use of `DEFINED ENV{MVPN_IOS_ADJUST_TOKEN}`